spec/controllers/admin/reports_controller_spec.rb

```diff
@@ -311,13 +311,14 @@
 
     it "triggers the delivery report" do
       spree_post :create, {
+        format: :turbo,
```

Comment:
I'm wondering what happens if a user `ctrl+click`s (`cmd+click` on mac) on the report button. Normally this submits the form to a new tab, which could be useful in some cases.
I'm guessing we get a template missing error. I think we could easily support that with a similar html template.

But how likely it is that someone would do that, I don't know (maybe I'm the only one who would ever try). So let's wait for Bugsnag to tell us ;)

Reply:
I never tried that on a button, only links. So let's wait and see. But having a fallback could be useful in the future and it may help when we replace websockets.